apps/examples.nameguard.io/package.json

```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "@headlessui-float/react": "0.11.4",
     "@headlessui/react": "1.7.17",
+    "@namehash/ens-utils": "1.12.1",
```

Comment:
@FrancoAguzzi Shouldn't we be importing this from workspace:* ? This package also lives in the monorepo